src/Model/ClearableManagerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Doctrine\Model;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ *
+ * @phpstan-template T of object
```

Comment:
I don't think you need this since it's not used.

Reply:
Copy paste mistake, fixed.